src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -54,6 +54,7 @@ class LlmDropdown(
     models.sortedBy { it.codyProOnly }.forEach(::addItem)
 
     CodyAuthenticationManager.getInstance(project).getActiveAccountTier().thenApply { accountTier ->
+      if (accountTier == null) return@thenApply
```

Comment:
don't we have to worry about the other place?

https://sourcegraph.com/-/editor?remote_url=github.com%2Fsourcegraph%2Fjetbrains&branch=main&file=src%2Fmain%2Fjava%2Fcom%2Fsourcegraph%2Fcody%2FMyAccountTabPanel.kt&start_row=98&start_col=4&end_row=102&end_col=5&editor=JetBrains&version=v5.5.6-nightly

Reply:
That function does not need to loop into itself, it will be called when the account will change:
```
    ApplicationManager.getApplication()
        .messageBus
        .connect()
        .subscribe(LafManagerListener.TOPIC, LafManagerListener { update() })
```

Reply:
In that case, that callback seems to be completely redundant. I'm going to remove it.